includes/Modules/Analytics_4.php

```diff
@@ -91,6 +91,13 @@ final class Analytics_4 extends Module
 	 */
 	const MODULE_SLUG = 'analytics-4';
 
+	/**
+	 * Custom dimensions tracked by Site Kit
+	 */
+	const CUSTOM_EVENT_PREFIX              = 'customEvent:';
```

Comment:
This prefix is specific to use in reports, it isn't applied globally (e.g. when tagging) so we may want to clarify that or move this to a report-specific class.